hypertrace-trace-enricher/trace-reader/src/test/java/org/hypertrace/trace/reader/entities/DefaultTraceEntityReaderTest.java

```diff
@@ -60,13 +60,17 @@
 
   private static final StructuredTrace TEST_TRACE = defaultedStructuredTraceBuilder().build();
   private static final Event TEST_SPAN = defaultedEventBuilder().setCustomerId(TENANT_ID).build();
-  private static final Entity EXPECTED_AVRO_ENTITY =
+  private static final Entity EXPECTED_ENTITY =
       Entity.newBuilder()
           .setEntityType(TEST_ENTITY_TYPE_NAME)
           .setEntityId(TEST_ENTITY_ID_ATTRIBUTE_VALUE)
           .setEntityName(TEST_ENTITY_NAME_ATTRIBUTE_VALUE)
-          .setCustomerId(TENANT_ID)
-          .setAttributes(
+          .putAllAttributes(
+              buildAttributesWithKeyValues(
+                  Map.of(
+                      TEST_ENTITY_ID_ATTRIBUTE_KEY, TEST_ENTITY_ID_ATTRIBUTE_VALUE,
+                      TEST_ENTITY_NAME_ATTRIBUTE_KEY, TEST_ENTITY_NAME_ATTRIBUTE_VALUE)))
+          .putAllIdentifyingAttributes(
```

Comment:
Preexisting issue it looks like, highlighted by the new change. Name should not be identifying, and in fact since this is working with v2 typed entities, identifying attributes shouldn't be used at all (which I guess is why this isn't a problem, they'll be ignored, but still incorrect).

Reply:
Didn't dive into code much, but I think it might be happening because of this

https://github.com/hypertrace/hypertrace-ingester/blob/main/hypertrace-trace-enricher/trace-reader/src/main/java/org/hypertrace/trace/reader/entities/DefaultTraceEntityReader.java#L95

Reply:
Right - line 99 is incorrect (we need not set any identifying attributes if we're setting an id), could you pull it out and update the test